src/Chr.Avro/Abstract/SchemaBuilder.cs

```diff
@@ -767,6 +777,20 @@ public override Schema BuildSchema(TypeResolution resolution, ConcurrentDictiona
     /// </summary>
     public class TimestampSchemaBuilderCase : SchemaBuilderCase
     {
+        private readonly TemporalBehavior _temporalBehavior;
```

Comment:
to match the serde builder cases, this should be a public readonly property (`public TemporalBehavior TemporalBehavior { get; }`)